ui/src/app/machines/views/AddMachine/AddMachineForm/AddMachineForm.js

```diff
@@ -107,13 +82,31 @@ export const AddMachineForm = () => {
     () => setSavingMachine(false)
   );
 
-  const MachineSchema = usePowerParametersSchema(
-    powerType,
-    generateMachineSchema
-  );
-
-  const allPowerParameters = useAllPowerParameters(powerTypes);
-
+  const initialPowerParameters = useInitialPowerParameters(powerTypes);
+  const AddMachineSchema = Yup.object().shape({
+    architecture: Yup.string().required("Architecture required"),
+    domain: Yup.string().required("Domain required"),
+    extra_macs: Yup.array().of(
+      Yup.string().matches(
+        /^([0-9A-Fa-f]{2}:){5}([0-9A-Fa-f]{2})$/,
```

Comment:
We should make this a constant somewhere.

Reply:
Done - added `app/base/validation/ts` with this regex